src/Simplex/Chat/Operators.hs

```diff
@@ -84,8 +94,9 @@ data ConditionsAcceptance
   deriving (Show)
 
 data ServerOperator = ServerOperator
-  { operatorId :: OperatorId,
+  { operatorId :: Maybe OperatorId,
```

Comment:
I don't think it's a good idea to have Maybe here just for population purposes, it's UI facing types and we'll be dealing with this optionality everywhere. Can have separate type ServerOperatorTemplate and function to transform it